CHANGES/6223.bugfix

```diff
@@ -0,0 +1 @@
+Including file type extension
```

Comment:
Can this call out the upload case as when the user would experience it?

Reply:
sure!